refs/iso19100.json

```diff
@@ -0,0 +1,293 @@
+{
+    "iso-19101-1": {"aliasOf": "iso-19101-1-2014"},
+    "iso-19101-1-2014": {
+        "authors": ["ISO/TC 211"],
+        "href": "https://www.iso.org/standard/59164.html",
+        "title": "Geographic information -- Reference model -- Part 1: Fundamentals",
+        "publisher": "ISO",
+        "deliveredBy": [{
```

Comment:
I'd forgo deliveredBy altogether unless you have a use case for it.

Reply:
The "deliveredBy" model is different in reference.json (list of url, shortname pairs) compared with raw-reference.json (list of URLs)  - is this the problem?

Our intention was to provide the link to the particular ISO Technical Committee which is responsible for all the ISO 19100 series standards. I think that is what deliveredBy is for?